perf/benchmark/run_benchmark_job.sh

```diff
@@ -44,23 +44,34 @@ function setup_metrics() {
 }
 
 function collect_metrics() {
-  local GENERATE_GRAPH=$1
-  local PLOT_METRIC=$2
-  CSV_OUTPUT="$(mktemp /tmp/benchmark_XXXX.csv)"
+  # shellcheck disable=SC2155
+  export CSV_OUTPUT="$(mktemp /tmp/benchmark_XXXX.csv)"
   pipenv install
   pipenv run python3 fortio.py $FORTIO_CLIENT_URL --csv_output="$CSV_OUTPUT" --prometheus=$PROMETHEUS_URL \
    --csv StartTime,ActualDuration,Labels,NumThreads,ActualQPS,p50,p90,p99,cpu_mili_avg_telemetry_mixer,cpu_mili_max_telemetry_mixer,\
 mem_MB_max_telemetry_mixer,cpu_mili_avg_fortioserver_deployment_proxy,cpu_mili_max_fortioserver_deployment_proxy,\
 mem_MB_max_fortioserver_deployment_proxy,cpu_mili_avg_ingressgateway_proxy,cpu_mili_max_ingressgateway_proxy,mem_MB_max_ingressgateway_proxy
+}
 
-  if [[ "$GENERATE_GRAPH" = true ]];then
-    BENCHMARK_GRAPH="$(mktemp /tmp/benchmark_graph_XXXX.html)"
-    pipenv run python3 graph.py "${CSV_OUTPUT}" "${PLOT_METRIC}" --charts_output="${BENCHMARK_GRAPH}"
-    dt=$(date +'%Y%m%d-%H')
-    RELEASE="$(cut -d'/' -f3 <<<"${CB_GCS_FULL_STAGING_PATH}")"
-    GRAPH_NAME="${RELEASE}.${dt}.${PLOT_METRIC}"
-    gsutil -q cp "${BENCHMARK_GRAPH}" "gs://$CB_GCS_BUILD_PATH/${GRAPH_NAME}"
-  fi
+function generate_graph() {
+  local PLOT_METRIC=$1
+  local OUTPUT_GRAPHS_DIR=$2
+  pipenv run python3 graph.py "${CSV_OUTPUT}" "${PLOT_METRIC}" --charts_output_dir="${OUTPUT_GRAPHS_DIR}"
+  gsutil -q cp -r "${OUTPUT_GRAPHS_DIR}" "gs://$CB_GCS_BUILD_PATH/${OUTPUT_GRAPHS_DIR}"
```

Comment:
When uploading perf_data to gcs bucket, let's remove the `tmp` dir prefix

Reply:
done